var/spack/repos/builtin/packages/py-deeptools/package.py

```diff
@@ -12,28 +12,17 @@ class PyDeeptools(PythonPackage):
 
     homepage = "https://pypi.io/packages/source/d/deepTools"
     # The test suite and associated test data is missing in the pypi tarball.
-    url      = "https://github.com/deeptools/deepTools/archive/3.3.0.tar.gz"
-
-    version('3.3.0', sha256='a7aaf79fe939ca307fe6ec5e156750389fdfa4324bf0dd6bf5f53d5fda109358')
-    version('3.2.1', sha256='dbee7676951a9fdb1b88956fe4a3294c99950ef193ea1e9edfba1ca500bd6a75')
-    version('2.5.2', sha256='16d0cfed29af37eb3c4cedd9da89b4952591dc1a7cd8ec71fcba87c89c62bf79')
+    url      = "https://files.pythonhosted.org/packages/f6/f3/789edda975fcca4736fab2007d82cab2e86739901c88bb0528db5c338d1f/deepTools-3.5.0.tar.gz"
```

Comment:
```suggestion
    pypi = "deepTools/deepTools-3.5.0.tar.gz"
```
You can also delete the two lines above this.

Reply:
Can you also re-add the versions you removed? You can do this with `spack checksum`